From c1ad450f7cde79149b73f25cc632d15830b75829 Mon Sep 17 00:00:00 2001
From: Louis Gesbert <louis.gesbert@ocamlpro.com>
Date: Tue, 21 May 2019 16:09:18 +0200
Subject: [PATCH 1/2] OCaml / static: fix linking options

---
 scripts/mk_util.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/scripts/mk_util.py b/scripts/mk_util.py
index 9076b582f..f076817d2 100644
--- a/scripts/mk_util.py
+++ b/scripts/mk_util.py
@@ -1958,6 +1958,7 @@
             print("WARNING: Could not find ocamlfind utility. OCaml bindings will not be installed")
 
     def mk_makefile(self, out):
+        global LDFLAGS
         if is_ml_enabled():
             CP_CMD = 'cp'
             if IS_WINDOWS:
@@ -2035,6 +2036,7 @@
             if is_cygwin() and not(is_cygwin_mingw()):
                 LIBZ3 = z3linkdep
 
+            LDFLAGS = LDFLAGS + ' -L.'
             LIBZ3 = LIBZ3 + ' ' + ' '.join(map(lambda x: '-cclib ' + x, LDFLAGS.split()))
 
             if DEBUG_MODE and not(is_cygwin()):
-- 
2.11.0

